README.md

```diff
@@ -1,6 +1,71 @@
 [![Build Status](https://travis-ci.org/openmrs/openmrs-module-calculation.svg?branch=master)](https://travis-ci.org/openmrs/openmrs-module-calculation)
 
-openmrs-module-calculation
-==========================
+# OpenMRS Calculation Module
+
+The Calculation module provides a framework for OpenMRS developers to author from simple to complex calculations to produce a particular piece of data for a single patient or a cohort of patients. It is a high level abstraction of rules/calculations, simple in that it provides a minimal number of useful concrete implementations of its core components that can be used out of the box but providing a much more flexible tool for authoring patient centered calculations.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the Calculation Module on a live system.
+
+### Prerequisites
+
+Follow instructions on OpenMRS web site to download and install the OpenMRS SDK, found at https://wiki.openmrs.org/display/docs/OpenMRS+SDK+Step+By+Step+Tutorials
+
+### Installing
+
+To install the Calculation Module you will need to download and install the module from github. Follow the following instructions to do this.
```

Comment:
Compiled version of the module are got from: https://addons.openmrs.org/#/show/org.openmrs.module.calculation